components/clarinet-sdk-wasm/src/core.rs

```diff
@@ -464,6 +464,10 @@ impl SDK {
             .get_data_var(&contract_id, var_name)
             .ok_or("value not found".into())
     }
+    #[wasm_bindgen(getter, js_name=getBlockTime)]
```

Comment:
shouldn't be a `getter` 

This is what causes the issue 
```
This expression is not callable.
  Type 'BigInt' has no call signatures.
```

Because getters are accessed without `()`